lib/pf/services/manager/haproxy.pm

```diff
@@ -58,6 +70,17 @@ sub generateConfig {
     $tags{'http'} = '';
     $tags{'mysql_backend'} = '';
     $tags{'var_dir'} = $var_dir;
+    my $bind_process = '';
+    if ($self->_number_cpus > 1) {
```

Comment:
If there is only one cpu it will be like now, only one process to handle the db and the portal.